src/controllers/companyController.js

```diff
@@ -0,0 +1,87 @@
+import { companies } from "../models/index.js";
+import notFoundError from "../erros/notFoundError.js";
+
+class CompanyController {
+  static async registerCompany(req, res, next) {
+    try {
+      const newCompany = await companies.create(req.body);
+      res.status(201).json({ message: "Loja criada!", company: newCompany });
```

Comment:
Segundo o contrato definido na task, em casos de sucesso o endpoint deve retornar apenas os dados da company, Fabinho. Nessa forma, não precisa da mensagem, apenas dos dados da company mesmo.
```suggestion
      res.status(201).json(newCompany);
```

Reply:
Seguirei mais a risca os contratos!